lib/src/code_generator/typealias.dart

```diff
@@ -173,4 +206,23 @@ class ObjCInstanceType extends Typealias {
     super.genFfiDartType,
     super.isInternal,
   }) : super._();
+
+  @override
+  String convertDartTypeToFfiDartType(
+    Writer w,
+    String value, {
+    bool objCShouldRetain = false,
+  }) =>
+      ObjCInterface.generateGetId(value, objCShouldRetain);
+
+  @override
+  String convertFfiDartTypeToDartType(
+    Writer w,
+    String value,
+    String library, {
+    bool objCShouldRetain = true,
+    String? objCEnclosingClass,
+  }) =>
+      ObjCInterface.generateConstructor(
+          objCEnclosingClass ?? 'NSObject', value, library, objCShouldRetain);
```

Comment:
Is `NSObject` ever happening?
If so, is it a sign we are not generating a class because it's not part of the `ffigen.yaml`?

If it can happen:
- Please add a test case that exercises this.
  - Leave a comment here to that test case.
- Emit a warning.

If it cannot (or should not) happen:
- Add an assert or exception.

Reply:
Shouldn't ever happen (clang should give an error). Added a null assertion instead.